MAUI/ProgressBar/States.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: States in .NET MAUI ProgressBar control | Syncfusion
+description: Learn here all about States support in Syncfusion Xamarin ProgressBar (Progress Bar) control and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---
+
+# States in .NET MAUI ProgressBar (Progress Bar)
+
+You can configure the states of the progress bar control depending on the usage.
+
+## Determinate
+
+This is the default state. You can use it when the progress estimation is known.
+
+## Indeterminate
+
+By enabling the `IsIndeterminate` property, the state of the progress bar can be changed to indeterminate when the progress cannot be estimated or is not being calculated. It can be combined with determinate mode to know that the application is estimating progress before the actual progress starts.
```

Comment:
By enabling the `IsIndeterminate` property, the state of the progress bar can be changed to indeterminate when the progress cannot be estimated or is not being calculated. It can be combined with a determinate mode to know that the application estimates progress before the actual progress starts.

Reply:
done.